packages/theme-chalk/src/scrollbar.scss

```diff
@@ -70,3 +59,16 @@
     }
   }
 }
+
+.el-scrollbar-fade-enter-active {
+  transition: opacity 340ms ease-out;
+}
+
+.el-scrollbar-fade-leave-active {
+  transition: opacity 120ms ease-out;
+}
```

Comment:
Where did you get the transition duration time?

Reply:
![image](https://user-images.githubusercontent.com/38392315/105657180-ca14d880-5efe-11eb-9c9a-1e6d60528e61.png)
The original sass